src/Entity/DeveloperCompaniesCacheInterface.php

```diff
@@ -41,6 +41,25 @@ interface DeveloperCompaniesCacheInterface {
    */
   public function getCompanies(string $id): ?array;
 
+  /**
```

Comment:
While this may work. I am not sure if this is the best approach.

We should create AppGroup Entities here.

Reply:
The above function only works for Edge and not for ApigeeX, for ApigeeX we have `getAppGroup()` function however  AppGroup caching has not yet implemented which is mentioned [here](https://github.com/apigee/apigee-edge-drupal/pull/825/files#diff-dce6159a29b9c2d2be06ba9fc3c7f60810512762696d33c4c14d20cee4b5c44fR282:~:text=it%20to%20the-,local,-cache%20so%20we) as TODO.

I will create a separate ticket for creating `AppGroup` entity for Edge module.

Reply:
Created ticket https://github.com/apigee/apigee-edge-drupal/issues/830 for above issue